tests/sentry/rules/processing/test_delayed_processing.py

```diff
@@ -562,8 +563,46 @@ def test_get_slow_conditions__only_fast(self):
 
 
 class ParseRuleGroupToEventDataTest(TestCase):
-    def test_parse_rulegroup_to_event_data(self):
-        pass
+    def setUp(self):
+        self.project = self.create_project()
+        self.group = self.create_group(self.project)
+        self.group_two = self.create_group(self.project)
+        self.rule = self.create_alert_rule(self.organization, [self.project])
+        self.rule_two = self.create_alert_rule(self.organization, [self.project])
+
+        self.event_data = {
+            "event_id": "1",
+            "occurrence_id": "1",
+        }
+
+        self.input_data = {
+            f"{self.rule.id}:{self.group.id}": json.dumps(self.event_data),
+            f"{self.rule_two.id}:{self.group_two.id}": json.dumps(self.event_data),
+        }
+
+        self.expected = {
+            (str(self.rule.id), str(self.group.id)): self.event_data,
+            (str(self.rule_two.id), str(self.group_two.id)): self.event_data,
+        }
+
+    def test_parse_rulegroup(self):
+        result = parse_rulegroup_to_event_data(self.input_data)
```

Comment:
nit - do we need to add the success case input_data to `self`?
Can we just define those here since we're not using them in other tests?

Reply:
👍 i did it first expecting more cases 🤣